website/blog/2018-01-10-1.10.0.md

```diff
@@ -0,0 +1,411 @@
+
+## Highlights
+
+### Introduce proper support for Vue Single File Components ([#3563]) by [@vjeux]
+
+There's a lot of demand for Vue SFC ([#2097]). We've introduced partial support for them: All the HTML is printed as is, but now the `<script>` and `<style>` tags are printed using Prettier.
+
+To use it, just run `prettier` on your `*.vue` files!
+
+### API
+
+#### Create the Prettier Plugin API ([#3536]) by [@azz]
+
+As Prettier for JavaScript has become stable, we've recently had contributors wanting to add new languages to Prettier, notably we had pull requests to add Python support and PHP support. We want to keep the core `prettier` package portable and maintainable, but we also want to give people the opportunity to run Prettier on more languages. To that end we've introduced a [plugin API]! Prettier plugins can contribute parsers and/or printers to the Prettier formatter. They are treated as first-class citizens and can even contribute to the embedding support (e.g. formatting your language inside Markdown).
+
+Using plugins is as simple as installing them via `npm`/`yarn`, and running Prettier as you usually would, there's no additional configuration to worry about!
+
+There are three official plugins under development:
+
+* [`@prettier/plugin-python`](https://github.com/prettier/prettier-python)
+* [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
+* [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
+
+All three of these plugins are still under active development and aren't _quite_ really to throw production code at, but keep an eye on them as they will be soon! If you're interested in helping build those plugins, check out their repository issues list or run it against your own code and start raising bugs! Similarly, if you're interested in building a new language in Prettier and need some help getting it off the ground, [create an issue in the prettier repo](https://github.com/prettier/prettier/issues/new) and let us help you out.
+
+Prettier's in-built languages have been refactored to be expressed in terms of the plugin API, this way we can guarantee going forward that the core API is generic.
+
+Check [the docs][plugin api] for more information.
+
+> Please note that as this is a new, significantly large feature, we're releasing this under a **beta** label in the documentation. While we don't anticipate anything significant, this means that we may make breaking changes in the next release.
+
+### Flow
+
+#### Print flow type annotation comments as comments ([#3449]) by [@duailibe]
+
+Flow [type annotation comments](https://flow.org/en/docs/types/comments/#toc-comment-type-annotation) are a nice way to get type checking without having to transpile your files. But before this release, Prettier would not print the type annotation as comments, making it impossible to use that feature; now Prettier will correctly detect if the type annotation was a comment and print accordingly.
```

Comment:
Not that it doens't work yet for `/*=` ones and this only works for the `flow`. In Babylon we also improved the position of various comments. cc @duailibe. So this is one step in the right direction and people should give us more examples where it's not working so we can fix it.

Reply:
@vjeux I'm aware #3493 :-) I thought about adding a "request for help" or something